README.md

```diff
@@ -9,53 +9,6 @@
 
 Kagome is a [Polkadot Runtime Environment](https://github.com/w3f/polkadot-spec/tree/master/runtime-environment-spec) developed by [Soramitsu](https://soramitsu.co.jp/) add funded by Web3 Foundation [grant](https://github.com/w3f/Web3-collaboration/blob/master/grants/grants.md). 
 
-## Supported features
```

Comment:
I think features should be in the beginning, so that everyone know what we are currently supporting

Reply:
I actually would remove them fully from the readme, this looks too much "draft".

Reply:
(e.g. use separate file, or "model" it into issues/milestones)

Reply:
okay, I am fine with it as long as we have a list of features